src/core/biblio.js

```diff
@@ -140,22 +140,34 @@ function bibref(conf) {
       </section>`;
     addId(sec);
 
-    refs.sort((a, b) =>
-      a.toLocaleLowerCase().localeCompare(b.toLocaleLowerCase())
-    );
-    const refObjects = refs.map(getRefContent);
+    const refObjects = { good: [], bad: [] };
+    refs.map(getRefContent).reduce((refObjects, ref) => {
```

Comment:
nit, this would be more consistent with the rest of the code as: 
```js
const { goodRefs, badRefs } = refs.map(getRefContent).reduce((refObjects, ref) => {
   // 
}, {goodRefs: [], badRefs: []});
```